tests/spread/general/configure-hook/task.yaml

```diff
@@ -0,0 +1,45 @@
+summary: snap configure hook test
+
+execute: |
+  # `rockcraft pull` will create the rockcraft project, buildd remote, and an instance
+  rockcraft init
+  rockcraft pull
+
+  # create an outdated instance that would have been created by craft-providers>=1.7.0<1.8.0
+  outdated_instance="base-instance-rockcraft-buildd-base-v00--be83d276b0c767e3ad60"
+  lxc launch --project rockcraft craft-com.ubuntu.cloud-buildd:core22 "$outdated_instance"
```

Comment:
I decided to test removing 1 outdated instance and 1 outdated image, rather than ALL the outdated images and instances.  The latter took over 20 minutes to execute :sob:.

Reply:
Was that because of the time it takes to launch the instances?

Reply:
Yeah, publishing (creating an image from an instance) was the most time consuming, launching was the second most consuming.